bin/smack.py

```diff
@@ -329,8 +329,11 @@ def svcomp_frontend(args):
   if len(args.input_files) > 1:
     raise RuntimeError("Expected a single SVCOMP input file.")
 
-  # test float benchmarks
-  if float_filter(args.input_files[0]):
+  # test float\bv benchmarks
+  filter_status = svcomp_filter(args.input_files[0])
+  if filter_status == 1:
```

Comment:
Shaobo, instead of returning integers 1 or 2, please return something more descriptive than that. This is Python and not C :). You can even return 'float' and 'bitvector' strings or something like that. Please fix this and pull again.